book/src/lint_configuration.md

```diff
@@ -805,3 +805,14 @@ for _ in &mut *rmvec {}
 * [`missing_errors_doc`](https://rust-lang.github.io/rust-clippy/master/index.html#missing_errors_doc)
 
 
+## `pub-underscore-fields-behavior`
+
+
+**Default Value:** `"PublicallyExported"`
+
+---
+**Affected lints:**
+* [`pub_underscore_fields`](https://rust-lang.github.io/rust-clippy/master/index.html#pub_underscore_fields)
+* [`or`](https://rust-lang.github.io/rust-clippy/master/index.html#or)
```

Comment:
I'm not really sure why this is here

Reply:
Me either! It's what `cargo collect-metadata` generated, other lints seem to have similar "Affected lints" sections :)

Reply:
I meant the `or` lint, that doesn't exist xD